libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -139,6 +139,12 @@ void ContractCompiler::appendInitAndConstructorCode(ContractDefinition const& _c
 		appendConstructor(*constructor);
 	else if (auto c = m_context.nextConstructor(_contract))
 		appendBaseConstructor(*c);
```

Comment:
I think so.  Otherwise the ``payable`` modifier on the base constructors have no effects.